TRRandomizerCore/Randomizers/TR1/TR1SecretRewardRandomizer.cs

```diff
@@ -49,9 +49,12 @@ private void RandomizeRewards(TR1CombinedLevel level)
         stdItemTypes.Remove(TR1Type.Pistols_S_P); // A bit cruel as a reward?
 
         int secretRoom = RoomWaterUtilities.DefaultRoomCountDictionary[level.Name];
```

Comment:
`secretRoom` looks to be unused now

Reply:
Oh yes, I wonder why VS didn't notice that.